ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -185,6 +185,16 @@
 {%     if router_bgp.address_family_evpn.route.import_match_failure_action is arista.avd.defined('discard') %}
 
 - VPN import pruning is __enabled__
+{%     endif %}
+{%     if router_bgp.address_family_evpn.neighbor_default_encapsulation.mpls is arista.avd.defined(true) %}
+#### EVPN Default Encapsulation
```

Comment:
Could we make this table more generic, so it will also work for vxlan use cases? If you modify the data model as suggested above, this should just output the value of `encapsulation` key.

Reply:
Fixed.

Reply:
Fixed.